.github/gbp.toml

```diff
@@ -7,6 +7,7 @@ reset_label = "GBP: Reset"
 "Atomic" = 2
 "Balance/Rebalance" = -5
 "Code Improvement" = 2
+"Documentation" = 2
```

Comment:
i think documentation should be less than code improvement

Reply:
thats okay with me